src/commands/ext-dev-register.ts

```diff
@@ -55,10 +57,14 @@ export const command = new Command("ext:dev:register")
         )}: ${err.message}`
       );
     }
-    return utils.logLabeledSuccess(
+    utils.logLabeledSuccess(
       logPrefix,
       `Publisher ID '${clc.bold(publisherId)}' has been registered to project ${clc.bold(
         projectId
+      )}. View and edit your profile at ${utils.consoleUrl(
+        projectId,
+        `/publisher/${publisherId}/dashboard`
```

Comment:
is publisherId needed?

I thought the URL is 
https://firebase.corp.google.com/project/extensions-staging-test/publisher/dashboard

Reply:
Can also drop `/dashboard` and rely on console redirect to decide which view to show by default, in case we change it in the future

Reply:
Huh, i think there are some redirects in place - the link used here works too. Will switch this to the simplest one though, good call.